{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2025-04-06 20:02:33,929+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2025-04-06 20:03:16,332+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2025-04-06 20:08:22,444+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2025-04-06 20:08:50,651+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-16,"message":"Could not authenticate the user","s":"error"}},"timestamp":"2025-04-06 20:10:34,251+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-300,"message":"Invalid symbol provided","s":"error"}},"timestamp":"2025-04-06 22:22:43,708+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-300,"message":"Invalid symbol provided","s":"error"}},"timestamp":"2025-04-06 22:26:39,540+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"range_to":"range_to cannot be 100 days greater than range_from for resolutions 1, 2, 3, 5, 10, 15, 20, 30, 45, 60, 120, 180 and 240"},"message":"Invalid input","s":"error"}},"timestamp":"2025-04-06 23:29:04,454+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:164] fyersModel","message":{"API":"/history","error":"HTTPSConnectionPool(host='api-t1.fyers.in', port=443): Max retries exceeded with url: /data/history?symbol=NSE%3ANIFTY2540923300CE&resolution=5&date_format=1&range_from=2025-03-01&range_to=2025-04-05&cont_flag=1 (Caused by NameResolutionError(\"<urllib3.connection.HTTPSConnection object at 0x000001F727F8B4D0>: Failed to resolve 'api-t1.fyers.in' ([Errno 11001] getaddrinfo failed)\"))"},"timestamp":"2025-04-07 02:53:29,331+0530","service":"FyersAPI"}
{"level":"ERROR","location":"[get_call:146] fyersModel","message":{"API":"/history","Error":{"code":-50,"data":{"date_format":"must be provided","from":"from range cannot be greater than to range","range_from":"must be provided","range_to":"must be provided","resolution":"must be provided","symbol":"must be provided"},"message":"Invalid input","s":"error"}},"timestamp":"2025-08-19 20:23:32,978+0530","service":"FyersAPI"}
